### PR TITLE
adding words to list

### PR DESCRIPTION
**Pull request by mattboch**

### CODE REVIEW

**Comment by mattboch** [comment]
having looked at how bad words works, I’m realizing now that some of the terms I’ve added aren’t needed,  as they contain substrings of other terms. I’ll make a new pull later with those issues resolved

File: lib/badwords.json [hunk 1]
```diff
@@ -1,61 +1,87 @@
...
+    "heeb",
     "homo",
     "hooker",
     "idiot",
     "insane",
     "insanitie",
     "insanity",
     "jap",
+    "kaffer",
+    "kaffir",
```
Comment (brainwane) [inline lib/badwords.json:47]: @dariusk  I mentioned this project on MetaFilter in [a discussion about automatically filtering for a certain slur](https://metatalk.metafilter.com/25598/Feature-Request-prevent-racial-slurs-from-being-posted). Per [this comment](https://metatalk.metafilter.com/25598/Feature-Request-prevent-racial-slurs-from-being-posted#1371843) I second the request for at least this line to be added to the list.

**Comment by dariusk** [comment]
Much belated addition of these words. Thanks all.

**Comment by brainwane** [comment]
Thank you!

**Comment by mattboch** [comment]
Thanks Darius! Sorry I totally blanked on resubmitting this with the substrings removed.

**Pull request merged**